package.json

```diff
@@ -43,18 +45,15 @@
     "@babel/core": "^7.8.4",
     "@carforyou/eslint-config": "3.0.5",
     "@carforyou/pika-plugin-build-components": "^1.0.9",
-    "@percy/storybook": "^3.2.0",
+    "@percy/storybook": "^3.3.1",
     "@pika/pack": "^0.5.0",
     "@pika/plugin-copy-assets": "^0.9.0",
     "@pika/plugin-standard-pkg": "^0.9.0",
     "@pika/plugin-ts-standard-pkg": "^0.9.0",
-    "@storybook/addon-actions": "^5.3.17",
-    "@storybook/addon-info": "^5.3.17",
-    "@storybook/addon-knobs": "^5.3.17",
-    "@storybook/addon-links": "^5.3.17",
-    "@storybook/addons": "^5.3.17",
-    "@storybook/react": "^5.3.17",
-    "@storybook/storybook-deployer": "^2.8.6",
```

Comment:
removing the deployer might break some of the scripts the build pipeline relies on. 
we wanted to change the storybook deployment anyway, see https://autoricardo.atlassian.net/browse/CFYEO-12 - i'm ok with an intermediary solution when we tackle this next. happy to help with it

Reply:
I re-installed the deployer package (before finding another solution)